Modules/_cursesmodule.c

```diff
@@ -1642,21 +1689,27 @@ static PyObject *
 _curses_window_enclose_impl(PyCursesWindowObject *self, int y, int x)
 /*[clinic end generated code: output=8679beef50502648 input=4fd3355d723f7bc9]*/
 {
+    // wenclose() returns always return a boolean
```

Comment:
I'm not sure that the comment is useful. Or do you mean that it cannot fail? I don't get it.

```suggestion
    // wenclose() always returns a boolean
```

Reply:
It cannot fail, according to the man. I'll actually drop those temporary comments aa they were mainly for reviewers to remember that this or that call doesn't need to be checked.